### DISCUSSION TITLE
Modelling propellers?

### DISCUSSION THREAD

**Discussion by cbosley** [turbinesFoam/turbinesFoam#348] [General]
Hi,

I need to be able to model both turbines and propellers in the same simulation. It's been a few months but I've found turbinesFoam to be helpful for modeling turbines. Is there a straightforward way to model a compressor? Would just negating all of the cl values in the appropriate elementProfiles be enough? I assume I will probably have to disable the end effects models.

**Comment by petebachant** [comment]
You should be able to take one of the turbine examples and negate the blade pitch or twist. Might get you there but I can't say for sure!
